internal/elasticsearch/client.go

```diff
@@ -461,3 +465,38 @@ func (c *Client) Indices(term string) ([]string, error) {
 
 	return ks, nil
 }
+
+func (c *Client) Delete(indices []string, requestBody map[string]interface{}) (*DeleteResult, error) {
+	endpoint := strings.Join(indices, ",") + "/_delete_by_query"
+
+	var buf bytes.Buffer
+	if err := json.NewEncoder(&buf).Encode(requestBody); err != nil {
+		return nil, err
+	}
+	body := bytes.NewReader(buf.Bytes())
+
+	c.logger.Info(buf.String())
```

Comment:
do we want to log these out? (and if you don't can probably just pass `&buf` to `doRequest`)

Reply:
Nope, leftover from debugging